glusterd2/transaction/transaction.go

```diff
@@ -133,6 +135,9 @@ func (t *Txn) Do() error {
 		}
 
 		if err := s.do(t.OrigCtx, t.Ctx); err != nil {
+			if t.DontCheckAlive && isNodeUnreachable(err) {
```

Comment:
Don't you have to continue here and not return ?

Reply:
yes, we need to continue. 
added continue as moved function after `t.DisableRollback` check